src/service/video/docking.js

```diff
@@ -724,6 +823,15 @@ export class VideoDocking {
     return {posY, posX: this.getRelativeX_()};
   }
 
+  /**
+   * @return {boolean}
+   * @private
+   */
+  isValidScrollingDirection_() {
+    return this.currentlyDocked_ ||
```

Comment:
Now only allows scrolling up for docking, while scrolling down works to undock as well.